src/world.js

```diff
@@ -231,6 +232,203 @@ $.extend( $.World.prototype, $.EventSource.prototype, /** @lends OpenSeadragon.W
         }
     },
 
+    /**
+     * Forces the system consider all tiles across all tiled images
+     * as outdated, and fire tile update event on relevant tiles
+     * Detailed description is available within the 'tile-invalidated'
+     * event.
+     * @param {Boolean} [restoreTiles=true] if true, tile processing starts from the tile original data
+     * @param {number} [tStamp=OpenSeadragon.now()] optionally provide tStamp of the update event
+     * @function
+     * @fires OpenSeadragon.Viewer.event:tile-invalidated
+     * @return {OpenSeadragon.Promise<?>}
+     */
+    requestInvalidate: function (restoreTiles = true, tStamp = $.now()) {
```

Comment:
I think it would make more sense for `TiledImage.requestInvalidate` to be the ground-source truth for this, and `World.requestInvalidate` to simply loop over the tiled images and call this on all of them.

Reply:
It is actually not that simple. You have to filter the tile cache to select tiles that belong to the image, only to discover in the end that you have to update all of them anyway. Unnecessary work. TiledImage also has its invalidation eimplementation, but it is not internally used - yet. I have some heuristics in mind, but for now we just update all the tiles we have.